src/event-bus.ts

```diff
@@ -0,0 +1,20 @@
+export interface IAddListener {
+  (key: string, callback: Function): void
+}
+
+export interface IRemoveListener {
+  (key: string, callback: Function): void
+}
+
+export interface IDispatch {
+  (key: string, context: any, value: any): void
+}
+
+export interface IEventBus {
+  addEventListener: IAddListener
+  removeEventListener: IRemoveListener
+  dispatch: IDispatch
+}
+
+const EventBus: IEventBus = require('eventbusjs')
```

Comment:
Fine with this lib and seems like it does everything we need so whatever, but did you consider using the node `EventEmitter` API instead? Would make it more standard for future contributors and potentially could even get a browser-native version eventually. There's a few browser implementations of this that we could pull in as a hard dependency (or wrap this lib in a small shim to make the API match).

Reply:
I'm not opposed to this, but our usage is so simplistic I'm not sure it's work the overhead of changing. We could always refactor.